frontend/src/App.scss

```diff
@@ -1,10 +1,93 @@
+@import "./styles/prelude.scss";
+
+.error {
+  color: red;
+}
+
 .App {
+  color: silver;
   padding: 20px;
   min-height: 100vh;
-  background-color: lightgray;
+  background-color: $primary-color;
   box-sizing: border-box;
+
+  h1, h3, h4 {
+    color: whitesmoke;
+  }
+
+  hr {
+    margin: 10px 0px 50px;
+  }
 }
 
-.error {
-  color: red;
-}
\ No newline at end of file
+.bi {
+  padding-right: 3px;
+}
+
+.bi-search {
+  color: silver;
+  padding-right: 5px;
+}
+
+.navBar {
+  display: flex;
```

Comment:
We have been using kebab case for our convention for class names, can you update this?